app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -160,7 +160,7 @@ def remote_execution_proxies(provider, *_rest)
         scoped_search relation: :pools, on: :pools_expiring_in_days, ext_method: :find_with_expiring_pools, only_explicit: true
 
         def add_back_cve_errors
-          if @pending_cve_attrs&.[](:content_view_id).present? || @pending_cve_attrs&.[](:lifecycle_environment_id).present?
+          if @pending_cve_attrs&.[](:content_view_id)&.present? || @pending_cve_attrs&.[](:lifecycle_environment_id)&.present?
```

Comment:
Hmm dont think this change is necessary. `nil.present? => false`

Reply:
```1] pry(main)> nil.present?
=> false
```
I guess present is not a standard ruby method  but its there in rails :)

Reply:
Ok, removed the commit